distribution.pm

```diff
@@ -101,8 +102,15 @@ sub script_run {
     # start console application
     my ($self, $name, $wait) = @_;
 
-    testapi::type_string "$name\n";
-    testapi::wait_idle($wait);
+    testapi::type_string "$name";
+    if ($wait > 0) {
```

Comment:
Please add check if `$wait` is defined. Else we will have undef warnings all over the place.